deploy.yaml

```diff
@@ -8,23 +8,25 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      name: enrober
+      component: enrober
   template:
     metadata:
       labels:
-        name: enrober
+        component: enrober
         routable: "true"
       annotations:
+        projectcalico.org/policy: "allow tcp from cidr 192.168.0.0/16; allow tcp from cidr 10.129.0.0/16"
         publicHosts: "test.k8s.local"
         publicPaths: "9000:/beeswax/deploy"
```

Comment:
I question whether we should even have a yaml file here since we have the other repo for that.